### PR TITLE
Adding Not available dialog

### PR DESCRIPTION
**Pull request by secondsun**
## Motivation

JIRA: https://issues.jboss.org/browse/AEROGEAR-3060

## Description

This displays a dialog for services that are not configured.

## Progress

- [x] Done Task

## Additional Notes

To verify create an empty mobile-services.json file then then try to navigate.  We do not have docs pages in the app yet so the docs button does nothing.

```
{
  "version": 1,
  "clusterName": "https://192.168.37.1:8443",
  "namespace": "myproject",
  "clientId": "app-android",
  "services": [

  ]
}


```

### CODE REVIEW

**Comment by secondsun** [comment]
@wei-lee Should we make the docs link go somewhere even though we don't have docs views yet?

File: app/src/main/java/com/aerogear/androidshowcase/navigation/Navigator.java [hunk 1]
```diff
@@ -92,12 +92,12 @@ public void navigateToAccessControlView(final BaseActivity activity) {
 
     public void navigateToStorageView(BaseActivity activity) {
         if (!isConfigured("notes-service")) {
-            showNotConfiguredDialog(activity, "notes-service");
+            showNotConfiguredDialog(activity, "notesservice");
```
Comment (danielpassos) [inline app/src/main/java/com/aerogear/androidshowcase/navigation/Navigator.java]: No space between words?

Comment (secondsun) [reply app/src/main/java/com/aerogear/androidshowcase/navigation/Navigator.java]: Fixed

**Pull request merged**